### PR TITLE
Attempting to fix package error

### PR DESCRIPTION
**Pull request by mrzengel**

### CODE REVIEW

File: .github/workflows/binder-on-pr.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name: Binder Badge
```
Comment (garciagenrique) [inline .github/workflows/binder-on-pr.yml:1]: You can remove this workflow file, I guess it was coming with `copier`.

All these test workflows are cool, but we are far away to make full usage of them

File: .github/workflows/check-release.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+name: Check Release
```
Comment (garciagenrique) [inline .github/workflows/check-release.yml:1]: Same here. we will implement automatic releases later.

File: .github/workflows/enforce-label.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+name: Enforce PR label
```
Comment (garciagenrique) [inline .github/workflows/enforce-label.yml:1]: Another workflow that can be removed.

File: .github/workflows/prep-release.yml [hunk 4]
```diff
@@ -0,0 +1,48 @@
+name: "Step 1: Prep Release"
```
Comment (garciagenrique) [inline .github/workflows/prep-release.yml:1]: This one too

File: .github/workflows/publish-release.yml [hunk 5]
```diff
@@ -0,0 +1,58 @@
+name: "Step 2: Publish Release"
```
Comment (garciagenrique) [inline .github/workflows/publish-release.yml:1]: as well as this one

File: .github/workflows/update-integration-tests.yml [hunk 6]
```diff
@@ -0,0 +1,51 @@
+name: Update Playwright Snapshots
```
Comment (garciagenrique) [inline .github/workflows/update-integration-tests.yml:1]: Also this one

**Review by garciagenrique — CHANGES_REQUESTED** [review]
The `/dist` folder should be ".gitignored" too

Otherwise LGTM! Thanks @mrzengel !!

**Comment by mrzengel** [comment]
I've .gitignored list/ and removed all template workflows

**Review by garciagenrique — APPROVED** [review]

**Pull request merged**